src/styles/variables/index.scss

```diff
@@ -0,0 +1,3 @@
+@import './tokens.scss';
```

Comment:
I believe the build is currently creating a css file for each of these individually.